qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/gsas2/test/test_gsas2_presenter.py

```diff
@@ -96,13 +96,13 @@ def test_get_limits_if_same_load_parameters(self):
         self.view.get_load_parameters.return_value = ["inst", "phase", "data"]
         self.assertEqual(self.presenter.get_limits_if_same_load_parameters(), None)
         # Success
-        self.view.get_x_limits_from_line_edits.return_value = [18000, 50000]
+        self.view.get_x_limits_from_line_edits.return_value = [["18000"], ["50000"]]
         self.view.get_load_parameters.return_value = ["inst", "phase", "data"]
-        self.assertEqual(self.presenter.get_limits_if_same_load_parameters(), [18000, 50000])
+        self.assertEqual(self.presenter.get_limits_if_same_load_parameters(), [["18000"], ["50000"]])
         # Success with limits reversed
-        self.view.get_x_limits_from_line_edits.return_value = [50000, 18000]
+        self.view.get_x_limits_from_line_edits.return_value = [["5000"], ["80000"]]
```

Comment:
It looks like the old test had the first element less than the second, and the new one doesn't - is that deliberate? I recall there were issues with the lower and upper limits getting swapped somewhere in the code, did you fix that here?

Reply:
`view.min_follow_mouse` and `view.max_follow_mouse` already make sure that xmin <= xmax for the cursors from the plot and revert them back to the original limits otherwise. As an extra safenet, I will keep the same logic by returning the sorted xmin and xmax.